tests/UI/campaigns/functional/BO/03_catalog/07_discounts/01_cartRules/01_CRUDCartRule/01_withAndWithoutCode.js

```diff
@@ -32,8 +32,20 @@ const {expect} = require('chai');
 let browserContext;
 let page;
 
+// Today date
+const now = new Date();
+// Day before
+const today = (`0${now.getDate()}`).slice(-2);
+// Current month
+const month = (`0${now.getMonth() + 1}`).slice(-2);
```

Comment:
I'm bad at JavaScript but maybe there is an easier way to get the month? https://www.w3schools.com/jsref/jsref_getmonth.asp